doc/ug/getting_started_build_sw.md

```diff
@@ -95,6 +95,159 @@ $ ./meson_init -t /path/to/toolchain/file
 If you do not specify your own toolchain configuration file (using `./meson_init.sh -t`), and `meson_init.sh` cannot find the default configuration in your toolchain, the legacy `toolchain.txt` from the main OpenTitan repository will be used.
 If `TOOLCHAIN_PATH` is set, this will be used to update any paths within the legacy configuration.
 
+## Building Software with Bazel
+
+The Mask ROM and many other software targets are built with
+(Bazel)[https://bazel.build/]
+
+TLDR:
+after installing Bazel you can run
```

Comment:
nit: colon at end of line (other lines should similarly end in either a colon or a full stop).